jira/jira-issues-to-dust.ts

```diff
@@ -235,7 +248,7 @@ function formatDescription(
 ): string {
   return (
     description?.content
-      .map((c) => c.content.map((cc) => cc.text).join(""))
+      .map((c) => c.content?.map((cc) => cc.text).join(""))
```

Comment:
Looks like some of our tickets add objects like `type: rule` in it, without any `content`.